int8 id
float32 tau
float32 dx
float32 x
